fe/fe-core/src/main/java/org/apache/doris/catalog/S3Resource.java

```diff
@@ -222,41 +226,36 @@ protected void getProcNodeData(BaseProcResult result) {
     public static Map<String, String> getS3HadoopProperties(Map<String, String> properties) {
         Map<String, String> s3Properties = Maps.newHashMap();
         if (properties.containsKey(S3_ACCESS_KEY)) {
-            s3Properties.put("fs.s3a.access.key", properties.get(S3_ACCESS_KEY));
+            s3Properties.put(Constants.ACCESS_KEY, properties.get(S3_ACCESS_KEY));
         }
         if (properties.containsKey(S3Resource.S3_SECRET_KEY)) {
-            s3Properties.put("fs.s3a.secret.key", properties.get(S3_SECRET_KEY));
+            s3Properties.put(Constants.SECRET_KEY, properties.get(S3_SECRET_KEY));
         }
         if (properties.containsKey(S3Resource.S3_ENDPOINT)) {
-            s3Properties.put("fs.s3a.endpoint", properties.get(S3_ENDPOINT));
+            s3Properties.put(Constants.ENDPOINT, properties.get(S3_ENDPOINT));
         }
         if (properties.containsKey(S3Resource.S3_REGION)) {
-            s3Properties.put("fs.s3a.endpoint.region", properties.get(S3_REGION));
+            s3Properties.put(Constants.AWS_REGION, properties.get(S3_REGION));
         }
         if (properties.containsKey(S3Resource.S3_MAX_CONNECTIONS)) {
-            s3Properties.put("fs.s3a.connection.maximum", properties.get(S3_MAX_CONNECTIONS));
+            s3Properties.put(Constants.MAXIMUM_CONNECTIONS, properties.get(S3_MAX_CONNECTIONS));
         }
         if (properties.containsKey(S3Resource.S3_REQUEST_TIMEOUT_MS)) {
-            s3Properties.put("fs.s3a.connection.request.timeout", properties.get(S3_REQUEST_TIMEOUT_MS));
+            s3Properties.put(Constants.REQUEST_TIMEOUT, properties.get(S3_REQUEST_TIMEOUT_MS));
         }
         if (properties.containsKey(S3Resource.S3_CONNECTION_TIMEOUT_MS)) {
-            s3Properties.put("fs.s3a.connection.timeout", properties.get(S3_CONNECTION_TIMEOUT_MS));
+            s3Properties.put(Constants.SOCKET_TIMEOUT, properties.get(S3_CONNECTION_TIMEOUT_MS));
         }
+        s3Properties.put(Constants.MAX_ERROR_RETRIES, "2");
         s3Properties.put("fs.s3.impl.disable.cache", "true");
-        s3Properties.put("fs.s3.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
-        s3Properties.put("fs.s3a.attempts.maximum", "2");
+        s3Properties.put("fs.s3.impl", S3AFileSystem.class.getName());
 
-        if (Boolean.valueOf(properties.getOrDefault(S3Resource.USE_PATH_STYLE, "false")).booleanValue()) {
-            s3Properties.put("fs.s3a.path.style.access", "true");
-        } else {
-            s3Properties.put("fs.s3a.path.style.access", "false");
-        }
+        s3Properties.put(Constants.PATH_STYLE_ACCESS, properties.getOrDefault(S3Resource.USE_PATH_STYLE, "false"));
+        s3Properties.put(Constants.AWS_CREDENTIALS_PROVIDER, properties.getOrDefault(S3Resource.S3_CREDENTIALS_PROVIDER,
+                TemporaryAWSCredentialsProvider.class.getName()));
```

Comment:
default should be `com.amazonaws.glue.catalog.credentials.ConfigurationAWSCredentialsProviderFactory`

Reply:
current use ConfigurationAWSCredentialsProviderFactory for glue